_search-plugins/hybrid-search.md

```diff
@@ -9,7 +9,7 @@ nav_order: 60
 Introduced 2.11
 {: .label .label-purple }
 
-Hybrid search combines keyword and neural search to improve search relevance. To implement hybrid search, you need to set up a [search pipeline]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/index/) that runs at search time. The search pipeline you'll configure intercepts search results at an intermediate stage and applies the [`normalization_processor`]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/normalization-processor/) to them. The `normalization_processor` normalizes and combines the document scores from multiple query clauses, rescoring the documents according to the chosen normalization and combination techniques. 
+Hybrid search combines keyword and neural search to improve search relevance. To implement hybrid search, you need to set up a [search pipeline]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/index/) that runs at search time. The search pipeline you'll configure intercepts search results at an intermediate stage and applies a [`score-based processor`]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/normalization-processor/) or [`rank-based processor`]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/rrf-processor/) to them. The processor normalizes and combines the document scores from multiple query clauses, rescoring the documents according to the chosen normalization and combination techniques. 
```

Comment:
Last sentence is from today's documentation and is focused on the score based processor. Can we incorporate there somehow that we do have two types now, rank and score based. Something along these lines: "The score-based processor normalizes and combines document scores, while the rank-based processor uses rank fusion to combine and re-rank the documents from multiple query clauses."